### ISSUE TITLE
Option for flickering when moving page

### ISSUE THREAD

**Issue by nateray42** [Guerra24/LRReader#44]
Hi, while I'm using the extremely helpful application, I have a suggestion for a trivial enhancement.

It would be cool if we could choose to flicker dark when we change a page.

When I move a page really fast, the fast flickering bright and dark makes me feel a bit tired. (also I need to wait a little when I navigate the page when it's loaded from dark)

If the feature is not labor intensive, choosing a simple page change mode would be cool (like, not involving flickering, but only changes page)

Below is the demonstration of the phenomena I stated.

https://github.com/user-attachments/assets/88a9429c-3485-4626-bc24-7b407fd960d6

**Comment by Guerra24** [comment]
Hello!

Do you mean disabling the fade animation?

**Comment by nateray42** [comment]
yes, thank you for addressing the proper word, it is the exact feature I intended.

**Comment by Guerra24** [comment]
v1.9.0 is now out with this change!
